account_financial_report/report/templates/general_ledger.xml

```diff
@@ -325,7 +327,7 @@
                                class="o_account_financial_reports_web_action"
                                style="color: black;">
                             <!--## We don't use t-field because it throws an error on click -->
-                                <t t-esc="line.date" t-options="{'widget': 'date'}"/></a>
+                                <t t-esc="line.date" t-options="{'widget': 'date', 'format': 'dd/MM/YY'}"/></a>
```

Comment:
The format date change depending of the country.
Use format date defined on language.

Reply:
How did you that ?
@fclementic2c do you want this to ?

Reply:
you can use https://github.com/OCA/server-tools/tree/11.0/datetime_formatter#usage if adding dependency is ok